napari/_app_model/_tests/test_viewer_toggler.py

```diff
@@ -15,6 +15,7 @@ def test_viewer_toggler():
     app = get_app()
     app.register_action(action)
 
+    # Injection required as there is no current viewer
```

Comment:
Yes I think there is no current viewer because we haven't used `make_napari_viewer`. I think I wanted to keep the test true to what Talley originally wrote so did not add `make_napari_viewer`.

We previously talked about using `make_napari_viewer` (instead of e.g., `init_qactions`, `initialize_plugins` in isolation) but I wonder if this is a good idea? It makes the tests less 'unit' as potentially there are interactions we are needlessly introducing (as sometimes we do not need all the things that are run in `make_napari_viewer`) ?

Note that here we do not need any of the setup functions ( `init_qactions` / `initialize_plugins`) to run.

Reply:
Yep, this seems fine to me @lucyleeow thanks for expanding on this.